.yaspellerrc.json

```diff
@@ -114,6 +114,7 @@
     "namespace",
     "objectId",
     "officiation",
+    "oldUrl",
```

Comment:
If everyone agrees with calling it oldUri or something more dsnpish wording rest looks good

Reply:
I prefer something like `target`. I don't think this should be a url, to me `contentUri` seems more compatible with existing spec.